src/cryptonote_core/service_node_list.cpp

```diff
@@ -842,6 +843,18 @@ namespace service_nodes
                               });
       if (cit != contributor.locked_contributions.end())
       {
+        if (hf_version >= cryptonote::network_version_19)
+        {
+          if (cit->amount < service_nodes::get_min_node_operator_contribution(node_info.total_contributed) && (block_height - node_info.registration_height) < service_nodes::SMALL_CONTRIBUTOR_UNLOCK_TIMER)
```

Comment:
Two comments:
- I don't think we should tie this necessarily to the operator minimum, but rather should have a separate constant.
- Passing in `node_info.total_contributed` doesn't seem right to me: if you have, for example, a still-open node with contributions of:
  - operator contributes 30%
  - staker contributes 10%
  then I think this is going to allow the staker to start an unlock (because they have 25% of the current contribution, but not the total required).
  
I think it would be easier to just have a single constant, something like `LARGE_STAKE_THRESHOLD` set to 3750 OXEN, and then here we can see if the amount is >= that threshold.

Actually I think we should make the threshold *slightly* lower, say 3749 OXEN, to avoid rounding errors in registration which sometimes slightly adjust a contribution value and might push it below the threshold.